dask/dataframe/io/parquet.py

```diff
@@ -416,6 +413,8 @@ def _write_partition_pyarrow(df, open_with, filename, write_index,
 
     if metadata_path is not None:
         with open_with(metadata_path, 'wb') as fil:
+            if 'compression' in kwargs:
+                kwargs.pop('compression')
```

Comment:
Not a big deal, but this might be slightly cleaner as either:

```python
if 'compression' in kwargs:
    del kwargs['compression']
```

or 

```python
kwargs.pop('compression', None)
```

I'd probably do the second, but I'll leave that up to you.